index.js

```diff
@@ -85,7 +99,16 @@ function setup(exercises) {
         fs.writeFileSync(item.dir + '/Readme.md', item.problem)
         fs.writeFileSync(item.dir + '/index.js', item.boilerplate)
       })
-    })
+      inquirer.prompt([{
```

Comment:
wish I had a better way to do this (press enter to continue). using [this](http://stackoverflow.com/a/19692588/146406) screws up subsequent terminal menu.